docs/index.md

```diff
@@ -28,10 +28,10 @@ After that we need a new virtualenv:
 
     python -m venv virtualenv
 
-Activate the virtualenv and then install strawberry
+Activate the virtualenv and then install strawberry plus the development server. 
```

Comment:
```suggestion
Activate the virtualenv and then install strawberry plus the debug server. 
```

Let's call it debug server, since the optional install is also called debug server :)